src/Digdir.Tool.Dialogporten.GenerateFakeData/DialogGenerator.cs

```diff
@@ -215,7 +215,7 @@ public static List<CreateDialogDialogActivityDto> GenerateFakeDialogActivities(i
             .RuleFor(o => o.CreatedAt, f => f.Date.Past())
             .RuleFor(o => o.ExtendedType, f => new Uri(f.Internet.UrlWithPath()))
             .RuleFor(o => o.Type, f => type ?? f.PickRandom<DialogActivityType.Values>())
-            .RuleFor(o => o.PerformedBy, f => GenerateFakeLocalizations(f.Random.Number(2, 4)))
+            .RuleFor(o => o.PerformedBy, f => f.Random.AlphaNumeric(10))
```

Comment:
```suggestion
            .RuleFor(o => o.PerformedBy, f => f.Name.FullName())
```

Dette skal være navn btw? 🤔